motd

```diff
@@ -31,5 +31,8 @@ echo "Connected to SSH tunnel server"
 echo
 printf "%s (%s %s %s)\n" "$DISTRIB_DESCRIPTION" "$(uname -o)" "$(uname -r)" "$(uname -m)"
 echo
+echo -e "HTTP   -->       \033[4;32mhttp://${USER}.cht-tunnel.plip.com\033[0m"
+echo -e "HTTPS  -->  \033[4;32mhttps://${USER}-ssl.cht-tunnel.plip.com\033[0m"
```

Comment:
I think this shouldn't be hard coded to a specific instance of this project.  We should leverage `installTunnelServer.sh` which gets the `DOMAIN.COM` [passed in](https://github.com/mrjones-plip/Keys-To-The-Tunnel/blob/main/installTunnelServer.sh#L3), and which we could in theory use to update the `motd` file when [we deploy it](https://github.com/mrjones-plip/Keys-To-The-Tunnel/blob/7c8678d59e75d48d5b9d878da44ef31c5e95c14e/installTunnelServer.sh#L100).

Reply:
:thinking: maybe `installTunnelServer.sh` can create a environment file with that variable set to make it available globally , eg. a file `/etc/environment.d/domain-kttt.conf` with a content like `DOMAIN=...`.

Reply:
hey @mrsarm ! I was [in here refactoring](https://github.com/mrjones-plip/Keys-To-The-Tunnel/pull/7) kttt and found a good solution to this I think.

During set up, we create a file called `say_my_name.sh` that echos back the host they're using:

```bash
#!/bin/bash
echo 'medic-tunnel.plip.com
```

Then in `motd` we can have this:

```bash
hostname=$(/root/Keys-To-The-Tunnel/say_my_name.sh)
echo "HTTP   -->     \033[4;32mhttps://$(logname).${hostname}\033[0m"
echo "HTTPS  --> \033[4;32mhttps://$(logname)-ssl.${hostname}\033[0m"
```

Works well!

```bash
Ubuntu 22.04.1 LTS (GNU/Linux 5.15.0-40-generic x86_64)

Connected to Keys-To-The-Tunnel

Press 'Enter' to exit

HTTP   -->     https://mrjones-plip.medic-tunnel.plip.com
HTTPS  --> https://mrjones-plip-ssl.medic-tunnel.plip.com
```

Thanks so much for the PR! It was a great idea that I'm happy to finally be implementing.  I'll put it in the other PR if you want to watch along!